### ISSUE TITLE
Major change published as minor if major disallowment is present in package. Expected or bug?

### ISSUE THREAD

**Issue by kevin-deyoungster** [microsoft/beachball#932]
Had a case where there was a `major` change file, but the `package.json` still had the disallowment 
```
"disallowedChangeTypes": [
      "major"
]
```

I expected publish to fail, but from pipeline logs, we see that the `major` change got published as a `minor`. Is this expected behavior?

**Comment by ecraig12345** [comment]
No, that's not expected--this might not be a case we've specifically thought about or handled, but I'd expect it to either fail when validating the change file, or successfully publish a major bump.
